blogposts/2021/how-to-deconstruct-the-monolith.md

```diff
@@ -0,0 +1,106 @@
+---
+title: 'How to deconstruct the monolith'
+author: Marcelo Oliveira
+authorUrl: 
+description: ''
```

Comment:
We really do need a description here 😆 
![Uploading Screenshot 2021-06-17 at 10.30.20.png…]()

Reply:
<img width="1146" alt="Screenshot 2021-06-17 at 10 30 20" src="https://user-images.githubusercontent.com/82806852/122402139-1f688980-cf75-11eb-82f6-34fb2c5bfa8c.png">
Sorry, here's the pic

Reply:
Thanks for the reminder! ;)

Reply:
@rebeccadee wdyt about these two options? I realize this description is super important to get people interested in clicking on the post from the blog homepage, so want to get this right! 

1. Breaking a monolith down into microservices can seem daunting. We rounded up the tools, patterns, and field-tested methods to guide your efforts towards a safe and consistent microservices development, so you have the knowledge you need to prepare for a successful migration.
2. Breaking a monolith down into microservices takes planning and deliberate work. We rounded up the tools, patterns, and field-tested methods to guide your efforts towards a safe and consistent microservices development.

Not married to either, or feel free to combine!

Reply:
```suggestion
description: 'Breaking a monolith down into microservices can seem daunting. We rounded up the tools, patterns, and field-tested methods to guide your efforts towards a successful migration.'
```